services/network/main.py

```diff
@@ -91,13 +93,25 @@ def get_saved_wifi_network(self):
 
         return WifiManager.__dict_from_table(data)
 
+    def hash_password(self, ssid, password):
+        """ This provides the same result as wpa_passphrase
+        """
```

Comment:
Remember to add arguments and the return type in the docstring

Reply:
done